### PR TITLE
BD-3160 B2B Docs section and Lead Scoring

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3160**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies.md [hunk 1]
```diff
@@ -27,5 +27,8 @@ guide_featured_list:
   - name: Sending Test Canvases
     link: /docs/user_guide/engagement_tools/canvas/testing_canvases/sending_test_canvases/
     image: /assets/img/braze_icons/user-check-01.svg
+  - name: Lead Scoring
+    link: /docs/user_guide/engagement_tools/canvas/testing_canvases/lead_scoring/
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies.md:31]: Looks like the incorrect subpage here
```suggestion
    link: /docs/user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring/
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+| **Increment By** or **Decrement By** | Enter the number of points that will be increased or decreased from the lead score.|
+{: .reset-td-br-1 .reset-td-br-2}
+
+### Step 5: Launch your Canvas
+
+That’s it! Your lead scoring Canvas is ready to launch.
+
+## External lead scoring
+
+Whether using one of our [technology partners]({{site.baseurl}}/partners/home/), your own internal lead scoring model, machine learning, or another lead scoring tool, we have multiple options for you!
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:114]: The preceding section uses an exclamation point with a celebratory tone. I would remove the one here to prevent too many in close succession.
```suggestion
Whether using one of our [technology partners]({{site.baseurl}}/partners/home/), your own internal lead scoring model, machine learning, or another lead scoring tool, we have multiple options for you.
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+### External partners
+
+Check out [Technology partners]({{site.baseurl}}/partners/home) to learn about our B2B partners that offer lead-scoring capabilities. Don’t see your partner there? You can integrate by calling our [`users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track#track-users) API endpoint.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:118]: They're our partners, but to our users, they're just software and tools. Consider a different term here.
```suggestion
Check out [Technology partners]({{site.baseurl}}/partners/home) to learn about our B2B partners that offer lead-scoring capabilities. Don’t see your tool there? You can integrate by calling our [`users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track#track-users) API endpoint. 
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+Check out [Technology partners]({{site.baseurl}}/partners/home) to learn about our B2B partners that offer lead-scoring capabilities. Don’t see your partner there? You can integrate by calling our [`users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track#track-users) API endpoint. 
+
+### Internal lead scoring data models
+
+You can integrate Braze with your internal data models, including lead scoring models, in various ways. See below for some common examples of how our customers have integrated with Braze.
+
+#### Integrated cloud data warehouse
+
+{% tabs %}
+{% tab Braze as a lead-scoring data source %}
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:127]: Consider simplifying these tab names. They are already quite long, and will likely expand with translation. Do we need them to be full sentences? What about removing "lead-scoring"?
```suggestion
{% tab Braze as a data source %}
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+{% endif %}
+```
+{% endraw %}
+
+{: start="5"}
+5. Include the following headers:
+
+| Header | Content |
+| --- | --- |
+| Authorization | {% raw %}`Bearer {{result.access_token}}`{% endraw %}. To retrieve a token, [configure a connected app for the OAuth 2.0 client credentials flow](https://help.salesforce.com/s/articleView?id=sf.connected_app_client_credentials_setup.htm&type=5) and then use Connected Content to retrieve the bearer from Salesforce: <br><br>{% raw %}<code>{% connected_content https://[instance].my.salesforce.com/services/oauth2/token <br>:method post <br> :body client_id=[client_id]&client_secret=[client_secret]&grant_type=client_credentials <br>:save result %}{% endraw %} <br> Bearer {% raw %}{{result.access_token}}</code>{% endraw %} |
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:183]: Two suggestions for this table:
- I'm worried users may copy the period at the end of the snippet. To remove it, I'd suggest making it its own line. I've added line breaks to do so.
- The link text for the Salesforce doc is very long, added a suggestion to simplify

```suggestion
| Authorization | {% raw %}`Bearer {{result.access_token}}`{% endraw %}<br><br>To retrieve a token, [configure a connected app](https://help.salesforce.com/s/articleView?id=sf.connected_app_client_credentials_setup.htm&type=5) for the OAuth 2.0 client credentials flow and then use Connected Content to retrieve the bearer from Salesforce: <br><br>{% raw %}<code>{% connected_content https://[instance].my.salesforce.com/services/oauth2/token <br>:method post <br> :body client_id=[client_id]&client_secret=[client_secret]&grant_type=client_credentials <br>:save result %}{% endraw %} <br> Bearer {% raw %}{{result.access_token}}</code>{% endraw %} |
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+| Content_Type | application/json |
+{: .reset-td-br-1 reset-td-br-2}
+
+![Webhook being composed with a Salesforce webhook URL, PATCH HTTP method, raw text request body, and request headers.][10]{: style="max-width:80%;"}
+
+#### Step 2b: Schedule webhook sends
+
+You’ll want the campaign to trigger anytime the user’s lead score changes. This campaign will trigger for any user whose score changes, but it will only affect users who aren't currently an MQL and have crossed the threshold you set in the previous step.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:191]: More of a voice suggestion, I don't think we should tell the user what they want. Let's rephrase this to either tell them what to do, or make it a recommendation.

```suggestion
The campaign should trigger anytime the user’s lead score changes. This campaign will trigger for any user whose score changes, but it will only affect users who aren't currently an MQL and have crossed the threshold you set in the previous step.
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+
+In the **Schedule Delivery** step, select the following:
+- An **Action-Based** delivery type
+- A trigger action of **Change Custom Attribute Value** with the name of your lead scoring attribute and an action of **any new value**
+
+#### Step 2c: Identify target audience
+
+In the **Target Audiences** step, include a filter that excludes users whose lead statuses are already at MQL or beyond, such as "`lead_status` is none of `MQL`".
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:199]: Technically I believe "is none of" should also be code text given that it is a selectable filter. I'm not sure if we've explicitly stated that anywhere. I like the separation with each filter name being opened and closed like it is, very clean. 👍 
```suggestion
In the **Target Audiences** step, include a filter that excludes users whose lead statuses are already at MQL or beyond, such as "`lead_status` `is none of` `MQL`".
```

**Review by bre-fitzgerald — COMMENTED** [review]
@rachel-feinberg looking good! Just went through and left some feedback for you. Also, it looks like the links on the Industry Use Cases page aren't functioning (I can't tell if the b2b landing page exists yet).

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+
+#### Step 4a: Add an Action Path
+
+Under your variant, select the plus icon and then select **Action Paths**.
+
+![Canvas with “Action Paths” displaying in the menu opened by the plus icon.][6]{: style="max-width:60%;"}
+
+#### Step 4b: Create Action Groups
+
+Each Action Group will represent all the actions that lead to the same point increment or decrement. You can set up to eight Action Groups. In this scenario, we'll setting up four groups.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:75]: Minor typo
```suggestion
Each Action Group will represent all the actions that lead to the same point increment or decrement. You can set up to eight Action Groups. In this scenario, we'll set up four groups.
```

File: _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+After your internal teams have created and run your lead scoring model, you can pull that data back into Braze so you can better segment and target leads for relevant messaging. You can do this with [Braze Cloud Data Ingestion]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/overview/). 
+
+With Cloud Data Ingestion, your internal teams will create a new table or view with your user identifiers, the latest lead scores, and the timestamps when the scores were updated. Braze will pick up the table or view and add the lead scores to the user profiles.
+
+{% endtab %}
+{% endtabs %}
+
+## Lead handoff: Marketing Qualified Lead (MQL) to Sales {#lead-handoff}
+
+Our recommended approach to lead handoffs is to have a corresponding lead or contact attached to each user in Braze. These leads would enter your sales teams’ queue when their lead statuses change to an MQL stage, at which point Salesforce would kick off a lead routing or assignment workflow.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/canvas/ideas_and_strategies/lead_scoring.md:148]:
```suggestion
Our recommended approach to lead handoffs is to have a corresponding lead or contact attached to each user in Braze. These leads would enter your Sales teams’ queue when their lead statuses change to an MQL stage, at which point Salesforce would kick off a lead routing or assignment workflow. 
```

**Review by lydia-xie — APPROVED** [review]
looks great! left some suggestions for your review!
